andela/bp-andela-media-frontend
# Create Header and Sidebar Styling

#### What does this PR do?
Introduces the header and sidebar required for future components.

#### Description of Task to be completed?
User should be able to view videos.

#### How should this be manually tested?
- Checkout to the branch
- Visit the route `header`
- Confirm that the view is in-line with this [mockup](https://projects.invisionapp.com/d/main/default/#/console/16015287/332140915/preview)

#### What are the relevant pivotal tracker stories?
[#162376936](https://www.pivotaltracker.com/story/show/162376936)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-12-11 at 15 03 38" src="https://user-images.githubusercontent.com/23451229/49818832-0d8a4800-fd74-11e8-9b37-345bf6ca0ead.png">
<img width="316" alt="screen shot 2018-12-11 at 15 04 08" src="https://user-images.githubusercontent.com/23451229/49818834-0d8a4800-fd74-11e8-8583-0a13155a2434.png">
